.circleci/config.yml

```diff
@@ -33,3 +32,48 @@ jobs:
       - run:
           name: Run extended author test
           command: dzil xtest
+  release:
+    docker:
+      - image: regentmarkets/debian-pg:stretch-5.26.2-96
+    steps:
+      - checkout
+      - *prepare
+      - *start_redis
+      - *setup_env
```

Comment:
Why do we start a redis instance and install dependencies during the release job ?

Reply:
because dzil will run test before release, and test need a redis instance. A better way is to test if there is redis and skip test if necessary. Or mock those redis calls. but that's out of this card scope.